docs/administration/fabrics/prophecy-fabrics/connections/s3.md

```diff
@@ -16,3 +26,32 @@ To create a connection with your Amazon S3 buckets, enter the following paramete
 | Secret Access Key ([Secret required](docs/administration/secrets/secrets.md)) | Your AWS secret access key                                     |
 | Region                                                                        | AWS region where your S3 bucket is located (e.g., `us-east-1`) |
 | Bucket Name                                                                   | Name of your S3 bucket                                         |
+
+## S3 permissions
+
+When you create an S3 connection in Prophecy, access permissions are tied to the credentials you use. This means you will only see the files and folders your S3 credentials have permission to access. Any actions you perform—such as reading or writing files—are done using those credentials.
+
+To fully leverage an S3 connection in Prophecy, you need the following S3 permissions:
+
+- Example
+- Example
+
+## Sharing connections within teams
+
+Connections in Prophecy are stored within [fabrics](docs/administration/fabrics/prophecy-fabrics/prophecy-fabrics.md), which are assigned to specific teams. Once an S3 connection is added to a fabric, all team members that have access to the fabric can use the connection in their projects. No additional authentication is required—team members automatically inherit the access and permissions of the original connection.
+
+:::caution
+Be mindful of the access level granted by the stored credentials. Anyone on the team will have the same permissions—including access to sensitive data if allowed.
+
+To manage this securely, consider creating a dedicated fabric and team for high-sensitivity connections. This way, only approved users have access to those credentials.
+:::
+
+## Sync connection
+
+Prophecy fetches data from S3 connections in the following ways:
+
+- When you browse an S3 connection in the [Environment browser](/analysts/pipelines), Prophecy fetches data on demand as you expand folders. You can manually refresh the Environment browser to see updated files.
+
+- When a pipeline runs, Source gems will read the latest available version of the data. Keep in mind that schema evolution may or may not be picked up automatically depending on the type of Source gem used.
```

Comment:
This last line seems a little vague and unhelpful. Maybe we can rephrase it to something like:  if the schema of the data stored in S3 was changed, you will need to do "Infer Schema" again from the Source gem to get the updated data.

Please rephrase as required